electronics_lib/Ws2812b.py

```diff
@@ -0,0 +1,66 @@
+from electronics_abstract_parts import *
+from .JlcPart import JlcPart
+
+
+class Ws2812b(DiscreteChip, FootprintBlock, JlcPart):
+    def __init__(self) -> None:
+        super().__init__()
+        self.vdd = self.Port(VoltageSink(
+            voltage_limits=(3.7, 5.3) * Volt,
+            current_draw=(0.6, 0.6 + 12*3) * mAmp)
+        )
+        self.gnd = self.Port(Ground())
+        self.din = self.Port(DigitalSink.from_supply(
+            self.gnd, self.vdd,
+            voltage_limit_tolerance=(-0.3, 0.7),
+            input_threshold_abs=(1.5, 2.3))
```

Comment:
```suggestion
            input_threshold_abs=(0.7, 2.7))
```

Linked datasheet says 0.7 for low input and 2.7 for high input?

I don't remember if we were taking into account the more restrictive limits of another part, but if we were we might want to note that as a comment.